rpcs3/Emu/Cell/lv2/sys_mmapper.cpp

```diff
@@ -218,7 +218,7 @@ error_code sys_mmapper_free_address(u32 addr)
 		return {CELL_EINVAL, addr};
 	}
 
-	if (!area.unique())
+	if (area.use_count() != 1)
```

Comment:
What was the warning with unique()?

Reply:
warning C4996: 'std::shared_ptr<vm::block_t>::unique': warning STL4016: std::shared_ptr::unique() is deprecated in C++17. You can define _SILENCE_CXX17_SHARED_PTR_UNIQUE_DEPRECATION_WARNING or _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS to acknowledge that you have received this warning.